{"name": "insights.specs.Specs.lpfc_max_luns", "exec_time": 5.316734313964844e-05, "errors": ["Traceback (most recent call last):\n  File \"/var/lib/insights/newest.egg/insights/core/plugins.py\", line 101, in invoke\n    return self.component(broker)\n  File \"/var/lib/insights/newest.egg/insights/core/spec_factory.py\", line 709, in __call__\n    return self.kind(\n  File \"/var/lib/insights/newest.egg/insights/core/spec_factory.py\", line 213, in __init__\n    self.validate()\n  File \"/var/lib/insights/newest.egg/insights/core/spec_factory.py\", line 218, in validate\n    raise ContentException(\"%s does not exist.\" % self.path)\ninsights.core.exceptions.ContentException: /sys/module/lpfc/parameters/lpfc_max_luns does not exist.\n"], "results": null, "ser_time": 1.0251998901367188e-05}